docs/releases/1.9.txt

```diff
@@ -483,6 +483,7 @@ CSRF
 
 * The request header's name used for CSRF authentication can be customized
   with :setting:`CSRF_HEADER_NAME`.
+* The setting :setting:`CSRF_TRUSTED_ORIGINS` was added to allow cross origin unsafe (e.g. `POST`) requests under HTTPS.
```

Comment:
need double backticks around POST to have an effect on the rendered docs (also wrap at 79).